static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -147,6 +148,8 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
   }>(jsFrameworkSelectOptions[0]);
 
   const defaultTab = 'npm';
+  const {location} = useRouteContext();
+  const crashReportOnboarding = location.hash === '#crashreport-sidequest';
```

Comment:
how about `#crashreport-sidequest` also goes into a `const` so we can refer to it here and on a couple of spots on `static/app/components/feedback/useFeedbackOnboarding.tsx` ?